screens/WalletDetailScreen.tsx

```diff
@@ -68,6 +77,26 @@ const IconImage = styled(Image)`
   overflow: hidden;
 `;
 
+const EyeIconImage = styled(Image)`
+  width: 25;
+  height: 25;
+  overflow: hidden;
+  left: 5;
+`;
+
+const VisibilityArea = styled(View)`
+  flex-direction: row;
+  justify-content: flex-end;
+  right: 15;
```

Comment:
Prefer multiples of 8 for spacing consistency.  `16` is used for left/right edge margins throughout the app, so in this case, use `16`.  
Purpose of the edge margins are for android phones with rounded edges to the screens.